### PR TITLE
Apply filters to core updates

### PR DESCRIPTION
**Pull request by NickGreen**
#### Changes in PR

- Apply time-based filter to core updates as well
- Send auto core update emails to `concierge@` email address

#### Notes

The major change to consider here is that we are turning on automatic core updates during business hours. I'm not sure if these are on by default for our sites already, but this would turn them on. Open to opinions here.

cc @mdrovdahl

### CODE REVIEW

**Comment by mdrovdahl** [comment]
> The major change to consider here is that we are turning on automatic core updates during business hours. I'm not sure if these are on by default for our sites already, but this would turn them on.

I'm 99% certain that WP Core updates on Pressable (as a client of WP Cloud) are pushed whenever the Atomic team updates WP Core in the underlying "WP Cloud" environment...which I think is an ad-hoc/whenever sort of matter. 

TL;DR I don't think this change will have a material impact on when Core updates are applied to sites hosted on Pressable or WP.com on Atomic.

**Comment by NickGreen** [comment]
> I'm 99% certain that WP Core updates on Pressable (as a client of WP Cloud) are pushed whenever the Atomic team updates WP Core in the underlying "WP Cloud" environment...which I think is an ad-hoc/whenever sort of matter.

TIL! In that case, it's possible that the auto-update email will not fire, since that doesn't fire when something is manually updated. 

However, we could still push these specific changes, and if they don't get you the emails that you want, we could consider using `wp_mail` to send something to us when core is updated for any reason, using:

`do_action( '_core_updated_successfully', $wp_version );`

**Comment by mdrovdahl** [comment]
Hmm...interesting. Yes, please do push this out and we'll see if we get any emails. If not, perhaps something we'll want to bring up with the Atomic team.

**Comment by NickGreen** [comment]
Done! Give it a couple of days for sites to update the plugin, and we'll see if anything comes of it.
